src/iframe.ts

```diff
@@ -19,7 +19,12 @@ type iAdvizeInterfaceParametersInternals = {
 type iAdvizeInterfaceParametersActivate = {
   command: 'internals';
   method: 'activate';
-  args: Record<string, unknown>;
+  args: {
+    authenticationOption: {
+      type: 'SECURED_AUTHENTICATION' | 'ANONYMOUS_AUTHENTICATION';
+      token: string;
+    };
```

Comment:
Pas de token pour l'anonyme, du coup on pourrait faire : 

```suggestion
 authenticationOption: 
   | {
      type: 'ANONYMOUS_AUTHENTICATION';
    } 
   |  {
      type: 'SECURED_AUTHENTICATION';
      token: string;
    };
```